html/script.js

```diff
@@ -2159,7 +2159,9 @@ function displayPhoto() {
         return;
     }
     let photos = SelectedPlane.psAPIresponse["photos"];
-    if (!photos || photos.length == 0) {
+    console.log(photos)
+    if (!photos || photos.length === 0 || photos.size === 0) {
```

Comment:
why check .size, it's an array.

Reply:
i suppose you could check typeof if you'd really want to go belt and braces.
And i hate using === when it's not necessary.
I've often seen it as === null which then doesn't catch === undefined while == null catches null and undefined ... which is usually the behaviour you want.